00_ES6/07_jest_test_reducer/README.md

```diff
@@ -0,0 +1,118 @@
+# Testing reducers
+
+The goal of this sample is to test a reducer using Jest.
+
+# Steps
+
+- Let's test `session-info` reducer. Start creating a simple spec file besides it and add a basic test just to make sure it is working:
+
+_./src/reducers/session-info.spec.js_
+```diff
```

Comment:
Same as in previous sample, use only diff when is a diff, if not javascript to get sintax color

Reply:
Sure. Modified and committed.